lib/puppet_x/certs/keystore.rb

```diff
@@ -0,0 +1,59 @@
+module Puppet_X
```

Comment:
I assume the `Puppet_X` name is a placeholder?

Reply:
This is a namespace to differentiate it from the Puppet namespace and prevent clashing. From all my research, that is the suggested way to provide your own common code.